python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_sobol_confidence.py

```diff
@@ -0,0 +1,295 @@
+bootstrap_size = 500
+fo_sample, to_sample = ComputeBootstrapChaosSobolIndices(
+    X, Y, basis, total_degree, distribution, bootstrap_size)
+
+# %%
+def ComputeSobolIndicesConfidenceInterval(fo_sample, to_sample, alpha=0.95):
+    """
+    From a sample of first or total order indices, 
+    compute a bilateral confidence interval of level alpha. 
+
+    Estimates the distribution of the first and total order Sobol' indices 
+    from a KernelSmoothing estimation. 
+    Then computes a bilateral confidence interval for each marginal. 
+
+    Parameters
+    ----------
+    fo_sample: ot.Sample(n, dim_input)
+        The first order indices
+    to_sample: ot.Sample(n, dim_input)
+        The total order indices
+    alpha : float
+        The confidence level
+
+    Returns
+    -------
+    fo_interval : ot.Interval
+        The confidence interval of first order Sobol' indices
+    to_interval : ot.Interval
+        The confidence interval of total order Sobol' indices
+    """
+    dim_input = fo_sample.getDimension()
+    fo_lb = [0] * dim_input
+    fo_ub = [0] * dim_input
+    to_lb = [0] * dim_input
+    to_ub = [0] * dim_input
+    for i in range(dim_input):
+        fo_i = fo_sample[:, i]
+        to_i = to_sample[:, i]
+        beta = (1.0 - alpha) / 2
+        fo_lb[i] = fo_i.computeQuantile(beta)[0]
+        fo_ub[i] = fo_i.computeQuantile(1.0 - beta)[0]
+        to_lb[i] = to_i.computeQuantile(beta)[0]
+        to_ub[i] = to_i.computeQuantile(1.0 - beta)[0]
+
+    # Create intervals
+    fo_interval = ot.Interval(fo_lb, fo_ub)
+    to_interval = ot.Interval(to_lb, to_ub)
+    return fo_interval, to_interval
+
+# %%
+# Compute confidence intervals from the indices samples.
+fo_interval, to_interval = ComputeSobolIndicesConfidenceInterval(fo_sample, to_sample)
+
+# %%
+def ComputeAndDrawSobolIndices(
+    N, basis, total_degree, distribution, bootstrap_size=500, alpha=0.95):
+    """
+    Compute and draw Sobol' indices from a polynomial chaos based on a 
+    given sample size. 
+    Compute confidence intervals at level alpha from bootstrap. 
+    """
+    X = distribution.getSample(N)
+    Y = g(X)
+    fo_sample, to_sample = ComputeBootstrapChaosSobolIndices(
+        X, Y, basis, total_degree, distribution, bootstrap_size)
+
+    fo_interval, to_interval = ComputeSobolIndicesConfidenceInterval(
+        fo_sample, to_sample, alpha)
+
+    graph = ot.SobolIndicesAlgorithm.DrawSobolIndices(
+        input_names,
+        fo_sample.computeMean(),
+        to_sample.computeMean(),
+        fo_interval,
+        to_interval)
+    graph.setTitle(f"Sobol' indices - N={N}")
+    graph.setIntegerXTick(True)
+    return graph
+
+
+# %%
+# Plot the boostrap Sobol' indices mean and the confidence intervals.
+# The confidence length will shrink if we increase the initial sample size.
+graph = ComputeAndDrawSobolIndices(30, basis, total_degree, distribution)
+_ = otv.View(graph)
+
+otv.View.ShowAll()
```

Comment:
A comment to analyze the result may improve the content. Here is a suggestion.
```suggestion
otv.View.ShowAll()

# %%
# We see that the variable F has the highest sensitivity indices and that confidence intervals 
# do not change this conclusion. 
# The confidence intervals of the sensitivity indices of the variables L and I are similar 
# so that we cannot say that L or I is more significant than the other: both variables have 
# sensitivity indices which are close to one another.
# The least sensitive variable is E, but the confidence intervals do not cross the X axis. 
# Hence, there is no evidence that the Sobol' indices of E are zero. 
```